src/pages/PrivateNotes/PrivateNotesEditPage.js

```diff
@@ -91,7 +91,7 @@ function PrivateNotesEditPage({route, personalDetailsList, session, report}) {
         Keyboard.dismiss();
 
         // Take user back to the PrivateNotesView page
-        Navigation.goBack(ROUTES.HOME);
+        Navigation.goBack(ROUTES.PRIVATE_NOTES_VIEW.getRoute(report.reportID, route.params.accountID));
```

Comment:
This seems to be okay as we always want to show the view page and then edit page.

Reply:
Okay. So is current change is correct?